lib/rubocop/cop/mixin/ordered_gem_node.rb

```diff
@@ -30,14 +30,16 @@ def consecutive_lines(previous, current)
       def register_offense(previous, current)
         message = format(
           self.class::MSG,
-          previous: gem_name(current),
-          current: gem_name(previous)
+          previous: gem_name(current.first_argument),
+          current: gem_name(previous.first_argument)
```

Comment:
Is `#first_argument` invoked in all call sites? If so maybe it should be part of the method implementation? 🤔

Reply:
Since I don't want to increase private method in `RuboCop/Cop/OrderedGemNode` mixin module in order to search recursively for the gem name, I was cutting out `#first_argument` to callers.

However, it seems that duplication of `#first_argument` was not good. 
(I was torn between the two implementations 💦)

So, I updated this implementation.